### ISSUE TITLE
Documentation Bug; Explaining how to test the finger server.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3885]
|<img alt="orsenthil's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| orsenthil reported|
|-|-|
|Trac ID|trac#3885|
|Type|defect|
|Created|2009-06-19 08:39:47Z|

The finger tutorial is great. But it assumes that the user already knows what he is going to try. :)
Unfortunately, I was not so and only after coming across this tutorial, did I know that finger could be seen a good network utility and can be used to explain the concepts.

While reading the tutorial the first, I struggled to figure out how to try the finger server which is running.

Attached patch is the documentation fix. Please consider.

Attachments:

* [docs-3885.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/26e/26e3dccd5fe806c7a68f697842aeeac6b05a17b0/9c346571da3ebb8ec298acab0024ae07c4f50a54.diff) (880 bytes) - added by orsenthil on 2009-06-19 08:41:16Z - finger tutorial documentation fix.
* [use-telnet-to-test-finger-server.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/26e/26e3dccd5fe806c7a68f697842aeeac6b05a17b0/6dbc5eb12599b357478eaca882dd491596dbc3d4.patch) (1560 bytes) - added by exarkun on 2009-07-14 18:35:49Z - alternate doc patch

<details><summary>Searchable metadata</summary>

```
trac-id__3885 3885
type__defect defect
reporter__orsenthil orsenthil
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1245400787000000 1245400787000000
changetime__1248283089000000 1248283089000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

Good idea.  I think the placement could be improved slightly.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="orsenthil's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|orsenthil commented|
|-|-|

exarkun, I checked your alternate docs. Its perfectly fine.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

I noticed a small mistake in my patch, the port number in the example telnet transcript was 22.  I'll change it to 1079, then apply.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [27174]) Apply use-telnet-to-test-finger-server.patch 

Author: orsenthil, exarkun
Reviewer: exarkun, orsenthil
Fixes: [#3885](https://github.com/twisted/twisted/issues/3885)

Add some documentation to the introduction of the finger tutorial explaining
how telnet can be used to test the server being developed.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
